backend/data_handler_app/models.py

```diff
@@ -22,41 +22,29 @@ class Answer(models.Model):
     def __str__(self):
         return self.answer
 
-class ClientData(models.Model):
-    # id field is automatically generated by django, we
-    # can create seperate id's for daily visitors, monthly
-    # visitors, etc.
-    date = models.DateTimeField(auto_now_add=True)
-    start_time = models.TimeField(auto_now_add=True)
-    completion_time = models.TimeField(auto_now_add=True)
+class Languages(models.Model):
+    id = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
For consistency, I would make sure we use `created_timestamp` and `modified_timestamp` here since we used those in other classes